auslib/test/admin/views/base.py

```diff
@@ -53,10 +53,27 @@ def setUp(self):
         dbo.permissions.user_roles.t.insert().execute(username="bill", role="releng", data_version=1)
         dbo.permissions.user_roles.t.insert().execute(username="bill", role="qa", data_version=1)
         dbo.permissions.user_roles.t.insert().execute(username="bob", role="relman", data_version=1)
+        dbo.permissions.user_roles.t.insert().execute(username="julie", role="releng", data_version=1)
```

Comment:
I'm not sure to understand what test case Julie helps testing. (I have nothing against Julies 😉 , but I only see her added to `testUsers`)